### PR TITLE
Feature/navbar2

### PR DESCRIPTION
**Pull request by joviantomd**
- Add bootstrap CDN (CSS & Javascript)
- Change some detail on Navbar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Product</a>
+            </li>
+            <li class="nav-item active">
+                <a class="nav-link" href="#">Contact Us</a>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#">Nav 2</a>
+                <a class="nav-link" href="#">Abous Us</a>
```
Comment (Justintnwj) [inline index.html]: Please Change Abous to About

**Review by Justintnwj — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (joviantomd) [reply index.html]: Okey noted

Comment (joviantomd) [reply index.html]: Done.

File: index.html [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
 </head>
+
 <body style="background-color: green;">
```
Comment (IkaPurnama) [inline index.html]: Please change background color green to white

**Review by IkaPurnama — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (joviantomd) [reply index.html]: Done.

File: index.html [hunk 3]
```diff
@@ -1,25 +1,34 @@
...
 </head>
-<body style="background-color: green;">
-    <nav>
-        <div>
-            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Modi mollitia repudiandae doloremque error molestias necessitatibus voluptas qui, reprehenderit quia, sequi recusandae corrupti distinctio reiciendis eum culpa illum iusto assumenda ut!
-        </div>
-    </nav>
-    <nav class="navbar navbar-expand navbar-light bg-light">
+
+<body style="background-color: white;">
```
Comment (IkaPurnama) [inline index.html:12]: approve

**Review by IkaPurnama — APPROVED** [review]

**Review by Justintnwj — APPROVED** [review]

**Pull request merged**